app/assets/templates/panel/navbar.jst.jade

```diff
@@ -8,6 +8,8 @@ nav.top-bar(role="navigation", data-topbar='', data-options="mobile_show_parent_
         span
   section.top-bar-section
     ul.right
+      li
+        a(href="/orders/history")  Orders History
```

Comment:
WDYTA calling it `My Orders`?

Reply:
Yup, looks much better, "order history" might be mistakenly received as whole order history instead